src/modules/gimbal/output_rc.cpp

```diff
@@ -101,6 +122,22 @@ void OutputRC::_stream_device_attitude_status()
 
 	matrix::Eulerf euler(_angle_outputs[0], _angle_outputs[1], _angle_outputs[2]);
 	matrix::Quatf q(euler);
+
+	// Adjust the angles if stabilization is being applied to represent the gimbal attitude in earth frame
+	// to comply with Mavlink 2 GIMBAL_DEVICE_ATTITUDE_STATUS definition
+	vehicle_attitude_s vehicle_attitude;
+	matrix::Eulerf euler_vehicle{};
+
+	if (_vehicle_attitude_sub.copy(&vehicle_attitude)) {
+		euler_vehicle = matrix::Quatf(vehicle_attitude.q);
+	}
+
+	const matrix::Quatf q_vehicle(matrix::Eulerf(
+					      (_stabilize[0]) ? euler_vehicle(0) : 0.0f,
```

Comment:
So these stabilize flags are still set and used? I wasn't aware of that but I also haven't tested such a gimbal ever.

Reply:
yes, they are still being used. I am using this with a roll only gimbal to have a camera work in a similar fashion to a S.O.D.A. 3D (to take oblique and nadir photos at the same pass) with MAV_CMD_OBLIQUE_SURVEY.